docs/index.rst

```diff
@@ -10,7 +10,7 @@ R5py
 
    installation
    configuration
-   basic-usage.ipynb
+   notebooks/basic-usage.ipynb
```

Comment:
Now called this `notebooks` but it could also be called `examples`? I have seen this style to be used with many packages

Reply:
@christophfink : Still about this: Do you prefer calling the directory `notebooks` or `examples`? I'm okay with both.

Reply:
I think notebooks works fine; I don’t mind examples, either. No strong opinion on this

Reply:
Let's keep notebooks for now at least